api_tests/institutions/views/test_institution_user_metric_list.py

```diff
@@ -76,7 +76,7 @@ def populate_counts(self, institution, user, user2):
             department='Biology dept',
             public_project_count=6,
             private_project_count=5,
-        ).save()
+        ).save(refresh=True)
```

Comment:
Curious too, I am not a familiar with `refresh=True`. I see some `save` has it while others not.

Reply:
by default, elasticsearch updates are not made available immediately, but only at the next periodic "refresh" -- there needs to be a refresh after this test data is added and before the test queries are run, and this was the easiest way i found via elasticsearch-dsl (which passes this [?refresh=true query param to the index api](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-refresh.html))

(i'd previously added `refresh=True` only to the last doc, because only one refresh is really needed, but seemed to hit nondeterministic test failures where an earlier document wasn't present yet, so i just made it refresh on every doc to be safe -- hacky, but the performance impact is negligible in these tiny test indexes)